RAKE/RAKE.py

```diff
@@ -27,19 +27,34 @@ def FoxStopList():
     return FoxStopList.words()
 
 
-def load_stop_words(stop_word_file):
+def MySQLStopList():
+    from .stoplists import MySQLStopList
+    return MySQLStopList.words()
+
+
+def NLTKStopList():
+    from .stoplists import NLTKStopList
+    return NLTKStopList.words()
+
+
+def load_stop_words(stop_word_file, divide, delimiter):
     """
     Utility function to load stop words from a file and return as a list of words
     @param stop_word_file Path and file name of a file containing stop words.
     @return list A list of stop words.
     """
     stop_words = []
-    for line in open(stop_word_file):
-        for word in line.split():  # in case more than one per line
-            stop_words.append(word)
+    if divide:
+        for line in open(stop_word_file):
+            for word in re.split(delimiter, stopword_file):
+                if word != '' or ' ':
```

Comment:
I think python will interpret this as `if word != '' or True`, because a non-empty string is truthy. You'd need to do `if word != '' or word != ' '`, or what I prefer `if word not in ['', ' ']`, in order to get it to test word with both conditions.